s3.tf

```diff
@@ -184,7 +184,24 @@ data "aws_iam_policy_document" "bucket" {
 
 }
 
+# TODO(luke): uncomment this in a follow up commit to make terraform-aws-collection
+# more likley to avoid the terraform apply failure
+# (due to s3 bucket notification eventual consistency)
+#
+# resource "time_sleep" "wait_before_subscribing" {
+#   depends_on = [
```

Comment:
`depends_on` must refer to a specific resource or module address and cannot accept arbitrary expressions

Reply:
done